charter.html

```diff
@@ -378,6 +384,10 @@ <h3 id="external-coordination">External Organizations</h3>
 		  <dl>
 			<dt><a href="https://www.ietf.org">IETF</a></dt>
 			<dd>A number of IETF working groups are likely venues for standardization of protocol components that advertising features depend on and research groups are investigating issues that will feed into the designs this group will consider. The main group we will need to coordinate with is the <a href="https://datatracker.ietf.org/wg/ppm/">Privacy Preserving Measurement</a> group</a>
+      <dt><a href="https://www.iab.com/">Interactive Advertising Bureau (IAB)</a></dt>
+      <dd>The IAB works with the media and marketing industries to promotes the interactive advertising industry and develops technical standards and solutions.</dd>
+      <dt><a href="https://iabeurope.eu/">IAB Europe</a></dt>
+      <dd>The IAB Europe is an European-level association for the digital marketing and advertising ecosystem.</dd>
```

Comment:
I'm opposed to these as they are proposed.

Yes, these groups have valuable insights to offer, but they have been engaging as participants.  And engaging effectively.    If we take this path, we might also add Google or Microsoft as external organizations that might be engaged with.

It would make sense if this effort needed to be coordinated with some work that is ongoing at those institutions.  Is there a particular effort that we can identify?

Is the work at the World Federation of Advertisers similarly positioned? I know that they are doing work on reach and frequency that might relate to the efforts here.  There at least, we might point at a specific work stream that might merit coordination.  (Though I think that this work is largely orthogonal to their work.)

Reply:
The IAB Tech Lab has an ongoing [workstream](https://iabtechlab.com/working-groups/privacy-sandbox-task-force/) to collect and define the industry's needs from privacy-focused ads technologies.  (It's called the "Privacy Sandbox Task Force" right now, but they acknowledge it should be renamed more expansively.)  The work has been more focused on ad selection mechanisms so far, but I believe the measurement and attribution aspects are equally in scope.  Points of contact are Miguel Morales and Hillary Slattery.

I am agnostic as to whether or not any of the various IAB groups ought to be listed in the charter; I'm merely providing this information in response to Martin's question about a specific relevant effort there.

Reply:
I will say that coordination with the ongoing efforts around standardization of the Open RTB standard in the IAB Tech Lab seems like it would be absolutely necessary for the launch of any feature that would have a direct impact on how they work or on how they are commonly used? I... have my concerns about the Task Force and I do think that it's not a workstream that requires quite as much coordination as to merit a spot here on its own, but surely the Tech Lab's ownership of specific advertising technical standards does merit this placement. I agree with Michael's earlier comment in this regard, that it makes sense to be more precise and specify the Tech Lab over IAB general. 

In regards to IAB Europe, they do own the TCF/GPP consent framework and signal standard which, as much as I may personally dislike it, is currently inescapable and would have to be considered as part of the overall impact and use of technologies we propose to develop in PAT. 

I think these are both meaningfully different than say, Microsoft (which is not maintaining a standard outside of the context of their W3C work) or WFA which, as far as I know, is not maintaining any technical standards? 

Whether we find the IAB Tech Lab and EU's technical standard work for the advertising industry effective, it is present, in heavy use, and our work would have a significant impact on it in such a way that I would think consultation makes sense. For the same reason we might consider a separate PR that added Prebid who is the primary developer of the underlying technical systems we intend to interact with. I think those are reasonable standards for this list?

Reply:
While I have high confidence that the WG's activities will include discussions with some of the listed IAB groups, I am in full agreement with Martin that (1) these do not need to be included in the charter and (2) if their inclusion would trigger a new AC review then the downside of further delay surely outweighs any upside of mentioning them in the charter.